tick/linear_model/tests/serializing_test.py

```diff
@@ -0,0 +1,49 @@
+
+# License: BSD 3 clause
+
+
+import io, unittest
+import numpy as np
+
+import dill as pickle
```

Comment:
Cant' we use pickle directly?

Reply:
yep

Reply:
Yep means we can or we can't?

Reply:
Yep means I did but forgot one

Reply:
maybe you need to refresh?

Reply:
Oh ok sorry, it's indeed shown as outdated

Reply:
can and updated